SQL_BestPractice/SQL-Best-hw.sql

```diff
@@ -0,0 +1,11 @@
+SELECT 
+    working_area, avg(commission) AS AVG_COMMISION,
+    count(agent_name) AS COUNT_AGENT_NAME
+FROM
+    agents
+HAVING 

```

Comment:
Please check order of HAVING and GROUP BY clause again?
Should we move HAVING clause below GROUP BY clause?